### ISSUE TITLE
Updates beyond V98

### ISSUE THREAD

**Issue by ChaosNicro** [ungoogled-software/ungoogled-chromium-fedora#24]
Heya,
Sorry to bother you, I'm using Opensuse Leap 15.3 and due to the recent CVEs had to switch to flatpack since there are no updated RPMs for UC. Are RPM based distros still supported or would it be best to get the generic tar instead? 
Thank you for your work and best wishes.

**Comment by wchen342** [comment]
Yes, I will not be able to update the versions for some time. Please use [chromium-freeworld](https://koji.rpmfusion.org/koji/packageinfo?packageID=506) in RPMFusion instead in the meantime.

**Comment by ChaosNicro** [comment]
Thats alright. Though the only description I can find for chromium-freeworld is "chromium with freeworld codecs and widevine-support", are the google dependencies really removed from that version?

**Comment by wchen342** [comment]
No, it is just a build with all available codecs, etc.

**Comment by bocekm** [comment]
On Fedora I switched to the flatpak version https://flathub.org/apps/details/com.github.Eloston.UngoogledChromium. It works fine, I just needed to tweak policy settings (https://github.com/flathub/com.github.Eloston.UngoogledChromium/issues/56).

**Comment by git-Hmmm** [comment]
@wchen342 If you can't update then just say the team clearly you no longer want to maintain this, they will find someone else at least we users don't have to struggle for update.

**Comment by wchen342** [comment]
There is not exactly a team and technically the only clear leader is @Eloston, other people just comes and goes. Otherwise there is no other person I can inform.
